lib/plugin/instance.rb

```diff
@@ -337,6 +337,12 @@ def add_permitted_post_create_param(name, type = :string)
     end
   end
 
+  def add_permitted_post_update_param(attribute, &block)
+    reloadable_patch do |plugin|
+      ::Post.plugin_permitted_update_params[attribute] = { plugin: plugin, handler: block }
```

Comment:
This overwrites any existing attribute. Is that what you wanted? Two plugins couldn't use the same attribute?

Reply:
I guess we could push this into an array!

Reply:
I followed the pattern for `plugin_permitted_create_params

```
  def add_permitted_post_create_param(name, type = :string)
    reloadable_patch do |plugin|
      ::Post.plugin_permitted_create_params[name] = { plugin: plugin, type: type }
    end
  end
```